### PR TITLE
excluded .gw.go from coverage and consistent yaml naming

### PR DESCRIPTION
**Pull request by ajansari95**
## 1. Overview
- consistent naming for ymls
- excluded gateway files from  coverage report 

## 2. Implementation details
- renamed all ```yaml``` to ```yml```
- excluded ```*pb.gw.go``` from coverage report

## 3. How to test/use

just run the workflows.

## 4. Checklist

- [ ] Does the Readme need to be updated?

### CODE REVIEW

**Review by puneet2019 — APPROVED** [review]

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -31,6 +31,7 @@ jobs:
         run: |
           excludelist="$(find ./ -type f -name '*.go' | xargs grep -l 'DONTCOVER')"
           excludelist+=" $(find ./ -type f -name '*.pb.go')"
+          excludelist+=" $(find ./ -type f -name '*.pb.gw.go')"
```
Comment (Anmol1696) [inline .github/workflows/test.yml:34]: does not seem to work in the coverage report that was commented in the PR

**Review by Anmol1696 — CHANGES_REQUESTED** [review]

File: .github/workflows/test.yml (hunk 1 above)
Comment (Anmol1696) [reply .github/workflows/test.yml:34]: Screenshot 2022-08-24 at 5 59 24 PM

Comment (ajansari95) [reply .github/workflows/test.yml:34]: resolved

**Review by Anmol1696 — APPROVED** [review]
LGTM

**Pull request merged**